sample/AirshipSample/ios/AirshipSample/AppDelegate.h

```diff
@@ -1,10 +1,8 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
Why is our sample code copyright "Facebook"?

Reply:
autogenerated

Reply:
Well, thanks Facebook!